app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask, render_template
+from flask import Flask, render_template, request, jsonify, send_from_directory
 from pages.hwtests import allboards as ab
 from pages.pyadi.plots import gen_line_plot_html
 from models import boards as b
 from models import boot_tests as bt
+from models.db import DB
+from utility import url_gen, filter_gen, artifact_url_gen
+from urllib.parse import urlparse, unquote
 
 # from junit2htmlreport import parser
 
 app = Flask(__name__)
 
+JENKINS_SERVER = '10.116.171.86'
+JENKINS_PORT = None
+BASE_PATH='/constellation'
 
-@app.route("/")
+@app.route(BASE_PATH+'/', endpoint='hello_world_alt')
```

Comment:
This endpoint suppose to be 'hello_world_alt'?

Reply:
This was done before due to confusion in routing, particularly on resolving {{url_for('hello_world')}}. Since two routes is assigned to one endpoint (In this case the name of method being routed to - hello_world()), I added alternate name for that particular route to distinguished them properly during debugging and force hrefs to use this particular route from the other. However, I can do away with this now and remove alternate endpoint name since the code is already working fine even without the alternate name.